socorro/unittest/processor/test_mozilla_transform_rules.py

```diff
@@ -7,33 +7,37 @@
 from StringIO import StringIO
 
 from configman.dotdict import DotDict as CDotDict
-from mock import Mock, patch
+from mock import call, Mock, patch
 
 from socorro.lib.datetimeutil import datetime_from_isodate_string
 from socorro.lib.util import DotDict
 from socorro.processor.mozilla_transform_rules import (
-    ProductRule,
-    UserDataRule,
-    EnvironmentRule,
-    PluginRule,
     AddonsRule,
+    AuroraVersionFixitRule,
+    BetaVersionRule,
     DatesAndTimesRule,
-    JavaProcessRule,
-    OutOfMemoryBinaryRule,
-    ProductRewrite,
     ESRVersionRewrite,
-    PluginContentURL,
-    PluginUserComment,
+    EnvironmentRule,
     ExploitablityRule,
     FlashVersionRule,
-    Winsock_LSPRule,
-    TopMostFilesRule,
-    BetaVersionRule,
-    AuroraVersionFixitRule,
+    JavaProcessRule,
     OSPrettyVersionRule,
+    OutOfMemoryBinaryRule,
+    PluginContentURL,
+    PluginRule,
+    PluginUserComment,
+    ProductRewrite,
+    ProductRule,
+    SignatureGeneratorRule,
     ThemePrettyNameRule,
+    TopMostFilesRule,
+    UserDataRule,
+    Winsock_LSPRule,
 )
+from socorro.signature.generator import SignatureGenerator
+from socorro.unittest import WHATEVER
 from socorro.unittest.testbase import TestCase
+from socorro.unittest.processor import get_basic_config, get_basic_processor_meta
```

Comment:
I think the processor import might be supposed to go before `testbase`

Reply:
It should! I'll fix that.